riverdriver/riverpgxv5/internal/dbsqlc/river_job.sql

```diff
@@ -290,11 +297,10 @@ river_job_scheduled AS (
     WHERE river_job.id = jobs_to_schedule.id
     RETURNING *
 )
-SELECT count(*)
-FROM (
-    SELECT pg_notify(@insert_topic, json_build_object('queue', queue)::text)
-    FROM river_job_scheduled
-) AS notifications_sent;
+SELECT
+    queue,
+    scheduled_at
```

Comment:
What do you think about having this return just the full job row instead? There's a minor performance implication, but keeps things more consistent and requires fewer bespoke types in the driver machinery.